src/client/ua_client.c

```diff
@@ -50,10 +52,12 @@ UA_Client * UA_Client_new(UA_ClientConfig config) {
 
 static void UA_Client_deleteMembers(UA_Client* client) {
     UA_Client_disconnect(client);
-    UA_Connection_deleteMembers(&client->connection);
-    UA_SecureChannel_deleteMembersCleanup(&client->channel);
-    if(client->endpointUrl.data)
-        UA_String_deleteMembers(&client->endpointUrl);
+    UA_Connection_deleteMembers(client->connection);
```

Comment:
shall i invent a new delete function for connection and securechannel? At the moment there is only the "FreeConnectionCallback" for the server...